cms/tests/tests_from_pkcs7_crate.rs

```diff
@@ -141,3 +141,9 @@ fn cms_decode_signed_der() {
     // should match the original
     assert_eq!(reencoded_der_signed_data_in_ci, der_signed_data_in_ci)
 }
+
+#[test]
+fn cms_decode_ber() {
+    let ber_pkcs7 = include_bytes!("../tests/examples/ber_pkcs7.bin");
```

Comment:
There's already a checked-in example: `cms_ber.bin`, which is what `cms_der.bin` is derived from